code/game/gamemodes/vampire/vampire_powers/vamp_400.dm

```diff
@@ -35,4 +35,24 @@
 				human.setEarDamage(human.ear_damage + (10 / min(get_dist(L, H), 5)))
 			if(issilicon(L))
 				L.Weaken(4)
-				playsound(L, 'sound/machines/warning-buzzer.ogg', 50, 1)
\ No newline at end of file
+				playsound(L, 'sound/machines/warning-buzzer.ogg', 50, 1)
+
+/obj/effect/proc_holder/vampire/batswarm
+	name = "Bat Swarm"
+	desc = "Summon a swarm of hostile bats to delay pursuers or cause chaos."
+	human_req = TRUE
+	blood_cost = 150
+	cooldownlen = 400
+
+/obj/effect/proc_holder/vampire/batswarm/fire(mob/living/carbon/human/H)
+	if(!..())
+		return
+	H.visible_message("<span class='warning'>[H] summons a swarm of bats!</span>")
+	playsound(get_turf(H), 'sound/magic/Ethereal_Enter.ogg', 50, 1, -1)
+	for(var/i = 1, i <= amount_to_spawn, i++)
+		var/mob/living/simple_animal/hostile/bat/B
+		B = new /mob/living/simple_animal/hostile/bat(get_turf(holder.my_atom))
+		B.faction |= "Vampire"
+		if(prob(50))
+			for(var/j = 1, j <= rand(1, 3), j++)
+				step(C, pick(NORTH,SOUTH,EAST,WEST))
```

Comment:
Also I'm making a new branch for the project. Would you mind pring it to that one when I have it live?